src/errormsg.sml

```diff
@@ -9,6 +9,7 @@ sig
     exception Error
     val impossible : string -> 'a   (* raises Error *)
     val reset : unit -> unit
+    val incrementLine : unit -> unit
```

Comment:
Function -> Mainly for readability and consistency of the style used for handling nested comments
ErrorMsg -> see below